docs/manipulating.rst

```diff
@@ -79,31 +79,65 @@ Numpy slicing notation::
 This returns a new :class:`~spectral_cube.SpectralCube` object
 with updated WCS information.
 
-Extracting a subcube from a ds9 region
---------------------------------------
+.. _reg:
+
+Extracting a subcube from a DS9/CRTF region
+-------------------------------------------
+
+Starting with spectral_cube v0.2, you can use `DS9
+<http://ds9.si.edu/doc/ref/region.html>`_/`CRTF
+<https://casaguides.nrao.edu/index.php/CASA_Region_Format>`_ regions to extract
+subcubes. The minimal enclosing subcube will be extracted with a two-dimensional
+mask corresponding to the DS9/CRTF region.  `Regions
+<https://astropy-regions.readthedocs.io/en/latest/>`_ is required for region
+parsing. The CRTF region may also contain spectral information which helps to
+extract the cube in the spectral dimension as well.
 
-Starting with spectral_cube v0.2, you can use ds9 regions to extract subcubes.
-The minimal enclosing subcube will be extracted with a two-dimensional mask
-corresponding to the ds9 region.  `pyregion
-<http://leejjoon.github.io/pyregion/>`_ is required for region parsing::
+This extracts DS9 region(s) from ``file.reg`` file. The `~regions.read_ds9`
+parses the ds9 file and converts it to a list of `~regions.Region` object::
 
-    >>> region_list = pyregion.open('file.reg')  # doctest: +SKIP
-    >>> sub_cube = cube.subcube_from_ds9region(region_list)  # doctest: +SKIP
+    >>> import regions # doctest: +SKIP
+    >>> region_list = regions.read_ds9('file.reg')  # doctest: +SKIP
+    >>> sub_cube = cube.subcube_from_regions(region_list)  # doctest: +SKIP
+
+This extracts CRTF regions from ``file.crtf`` file. The `~regions.read_crtf`
+also returns a list of `~regions.Region` object::
+
+    >>> import regions # doctest: +SKIP
+    >>> region_list = regions.read_crtf('file.reg')  # doctest: +SKIP
+    >>> sub_cube = cube.subcube_from_regions(region_list)  # doctest: +SKIP
 
 If you want to loop over individual regions with a single region file, you need to convert the individual
 region to a shape list due to limitations in pyregion::
```

Comment:
Since we're no longer using `pyregion`, you should edit this to show that you *don't* need to create `pyregion.ShapeList` objects any more.